public/global.css

```diff
@@ -1,3 +1,4 @@
+/* Commenting out everything from before because all this CSS is seemingly out of date and should be removed
```

Comment:
Let's delete this, since it's in Git history and we can re-add it super easily. Let me know if you'd like me to show you how that works.